packages/forma-36-react-components/src/components/Note/Note.css

```diff
@@ -1,19 +1,32 @@
 @import 'settings/colors';
 @import 'settings/typography';
 @import 'settings/dimensions';
+@import 'settings/borders';
 
 .Note {
-  padding: var(--spacing-s);
-  padding-right: var(--spacing-l);
+  padding: var(--spacing-m);
   margin: 0;
-
   font-family: var(--font-stack-primary);
   font-weight: var(--font-weight-normal);
   line-height: var(--line-height-default);
   font-size: var(--font-size-m);
-  color: var(--color-text-mid);
+  color: var(--color-text-base);
   display: flex;
   flex-direction: row;
+  border-radius: var(--border-radius-medium);
+}
+
+.Note__title {
+  font-weight: var(--font-weight-medium);
+  font-size: var(--font-size-l);
+  margin-bottom: var(--spacing-xs);
+  color: var(--color-text-dark);
+  line-height: 18px; /* ugly hack to align icon with text, it will be fixed when changes in typography will be implemented */
+}
+
+.Note__content {
+  width: 100%;
+  line-height: 18px; /* ugly hack to align icon with text, it will be fixed when changes in typography will be implemented */
```

Comment:
Temporary and ugly, but I think more accurate than moving icon down 1 or 2 px 🤷‍♀️ It will be removed as soon as we will implement changes in Typography in v4.